packages/compass/src/main/index.ts

```diff
@@ -2,6 +2,9 @@ import '../setup-hadron-distribution';
 import './setup-csfle-library';
 import { app } from 'electron';
 import { handleUncaughtException } from './handle-uncaught-exception';
+import {initialize } from '@electron/remote/main';
+
+initialize();
```

Comment:
See https://github.com/electron/remote#migrating-from-remote

We might have to revisit that README when we upgrade to electron 14:

> Note: Since this is requiring a module through npm rather than a built-in module, if you're using remote from a sandboxed process, you'll need to configure your bundler appropriately to package the code of @electron/remote in the preload script. Of course, [using @electron/remote makes the sandbox much less effective](https://medium.com/@nornagon/electrons-remote-module-considered-harmful-70d69500f31).

> Note: In electron >= 14.0.0, you must use the new enable API to enable the remote module for each desired WebContents separately: require("@electron/remote/main").enable(webContents).

> In electron < 14.0.0, @electron/remote respects the enableRemoteModule WebPreferences value. You must pass { webPreferences: { enableRemoteModule: true } } to the constructor of BrowserWindows that should be granted permission to use @electron/remote.